src/app/layout.tsx

```diff
@@ -13,37 +14,35 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en">
-         <head>
+      <head>
         <meta property="og:title" content="Farcaster Explorer" />
-        <meta property="og:description" content="Explore event propogation on farcaster." />
+        <meta property="og:description" content="Explore event propagation on farcaster." />
         <meta property="og:image" content="/neynar.png" />
         <meta property="og:url" content="https://www.neynar.com" />
         <meta property="og:type" content="website" />
         <meta name="twitter:card" content="summary_large_image" />
         <meta name="twitter:title" content="Farcaster Explorer" />
-        <meta name="twitter:description" content="Explore event propogation on farcaster." />
+        <meta name="twitter:description" content="Explore event propagation on farcaster." />
         <meta name="twitter:image" content="/neynar.png" />
       </head>
-      <body className="relative h-screen flex flex-col items-center justify-center min-h-screen">
+      <body className="relative h-screen flex flex-col items-center min-h-screen">
         <Providers>
           <div className="absolute top-0 left-0 p-2">
             <Link href="https://www.neynar.com" target="_blank">
-              <img className="w-16 md:w-32" src={"/neynar.png"} />
+              <img className="w-16 md:w-32" src="/neynar.png" />
             </Link>
           </div>
-          <div className="absolute top-5 left-1/2 transform -translate-x-1/2 p-2">
+          <Card className="min-w-[55%] md:min-w-[40%] fixed top-5 left-1/2 transform -translate-x-1/2 p-4 mb-5 z-10">
```

Comment:
can you make this absolute instead of fixed

Reply:
expanding on this I know this wasn't good on mobile before 😬 , can you try to improve it?

Reply:
yeah for sure, can make those changes and work on the mobile restyling/add a screenshot after :D